Gems/ROS2/Code/Source/Lidar/LidarSensorConfiguration.h

```diff
@@ -29,6 +30,15 @@ namespace ROS2
 
         //! Update the lidar system features based on the current lidar system selected.
         void FetchLidarImplementationFeatures();
+        static constexpr size_t maxClass = 256;
+        [[nodiscard]] AZStd::array<AZ::Color, maxClass> GenerateSegmentationColorsLookupTable() const;
+
+        static constexpr size_t unknownClassId = 0;
+        static const AZStd::string unknownClassName;
+        static const AZ::Color unknownClassDefaultColor;
+        static constexpr size_t groundClassId = 1;
+        static const AZStd::string groundClassName;
+        static const AZ::Color groundClassDefaultColor;
```

Comment:
This entire feature should at least be wrapped into a separate structure

Reply:
I don't think this is necessary, it's part of the configuration and checks Is visible can handle this neatly